cmd/cloud.go

```
@@ -672,6 +672,217 @@ func updateDeploymentStatusCompleted(logger logger.Logger, apiUrl, token, deploy
+	for _, d := range deployments {
+		date, err := time.Parse(time.RFC3339, d.CreatedAt)
+		if err != nil {
+			errsystem.New(errsystem.ErrApiRequest, err, errsystem.WithContextMessage("Failed to parse deployment date")).ShowErrorAndExit()
+		}
+		var msg string
+		if len(d.Message) > 60 {
+			msg = d.Message[:57] + "..."
+		} else {
+			msg = d.Message
+		}
+		tags := strings.Join(d.Tags, ", ")
```

golangci-lint (1.64.8)

ineffectual assignment to tags

(ineffassign)